draft-pauly-oblivious-doh.md

```diff
@@ -446,15 +478,22 @@ Oblivious DoH aims to keep knowledge of the true query origin and its contents k
 clients. In particular, it assumes a Dolev-Yao style attacker which can observe all client queries,
 including those forwarded by oblivious proxies, and does not collude with target resolvers. (Indeed,
 if a target colludes with the network attacker, then said attacker can learn the true query origin
-and its contents.) Oblivious DoH aims to achieve the following confidentiality goals in the presence
-of this attacker:
+and its contents.) Moreover, Oblivious DoH assumes that Proxies and Targets are not malicious and
+they do not collude. Malicious Targets are out of scope because DNS recursive resolvers behind
+targets may reply with legitimate or bogus answers in the absence of DNSSEC, so Target malfeasance
+does not introduce a unique threat. Moreover, malicious Proxies are out of scope because clients must
+trust them to access Targets. Malicious Proxies could block or otherwise tamper with Oblivious DoH
+queries, though this is not different from a malicious network which blocks client access to a
+DoH server.
+
+Oblivious DoH aims to achieve the following confidentiality goals in the presence of this threat model:
 
 1. Queries and answers are known only to clients and targets in possession of the corresponding
 response key and HPKE keying material. In particular, proxies know the origin and destination
 of an oblivious query, yet do not know the plaintext query. Likewise, targets know only the oblivious
 query origin, i.e., the proxy, and the plaintext query. Only the client knows both the plaintext
 query contents and destination.
-2. Target resolvers cannot link queries from the same client in the absence of unique per-client
+1. Target resolvers cannot link queries from the same client in the absence of unique per-client
```

Comment:
Is this numbering change correct?

Reply:
Yeah, Markdown renders it correctly. It saves us from having to update the numbers if we swap things around.